### PR TITLE
난이도별 랜덤 / CI

### PR DESCRIPTION
**Pull request by Godsenal**
난이도별 랜덤과 CI 추가

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -19,27 +22,42 @@ export const writeSolvedFiles = (path, prevSolvedArray, currSolvedArray) => {
...
+};
+
 export const findRandomExtinctItems = (items, count) => {
   const targets = new Set();
-  while (targets.size < count) {
-    const value = items[Math.floor(Math.random() * items.length)];
-    targets.has(value) || targets.add(value);
+  const levelMap = mapValues(
+    groupBy(items, (item) => item.split("-")[1]),
+    (items) => sampleSize(items, 3)
```
[the remark this answers is not in the document]
Comment (Jonir227) [reply utils.js:37]: 요기 3이 아니라 count가 들어가야할것같아요

**Review by Jonir227 — COMMENTED** [review]
미친 퍼포먼스

File: ci.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    repo: context.repo.repo,
+    title: `${new Date().toISOString().split("T")[0]} 문제`,
+    body: `
+# 오늘의 문제
+
+${questions
+  .map((question) => {
+    const [id, level] = question.split("-");
+
+    return `- ${level.toUpperCase()}: [${question}](${getQuestionUrl(id)})`;
```
[the remark this answers is not in the document]
Comment (Godsenal) [reply ci.js]: Number는 어떠한지? https://github.com/Jonir227/beat-type-challenge/pull/3/commits/ee7c9d96c1b1fd433872eba8829ec3c537d54477#diff-af9f5b22041f4ed7a7da3d1cf3fb0fa3f623473b2209a63f05d4c1b25ebbc758R19

File: utils.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Godsenal) [reply utils.js]: 그건 제 잔상입니다만

https://github.com/Jonir227/beat-type-challenge/pull/3/commits/27d0fc06fe00feb8cdcfe9fe71cbd8228bc46160#diff-fda8f0e13abe3d5c5685ad895fa528dd6067c41b66b6ebc6d1de71ca9a704d8dR37

**Review by Jonir227 — APPROVED** [review]
LJGTM (LOOKS JONNA GOOD TO ME)

**Comment by Godsenal** [comment]
CI 안되면 또하면되죠?

**Pull request merged**